admin/templates/SilverStripe/Admin/Includes/LeftAndMain_MenuLogo.ss

```diff
@@ -1,6 +1,6 @@
 <div class="cms-logo">
-    <a href="$ApplicationLink" target="_blank" title="$ApplicationName (Version - $CMSVersion)">
+    <a href="$ApplicationLink" class="cms-logo__link" target="_blank" title="$ApplicationName (Version - $CMSVersion)">
 		$ApplicationName <% if $CMSVersion %><abbr class="version">$CMSVersion</abbr><% end_if %>
     </a>
-    <span><% if $SiteConfig %>$SiteConfig.Title<% else %>$ApplicationName<% end_if %></span>
+    <span><a href="$BaseHref" target="_blank"><% if $SiteConfig %>$SiteConfig.Title<% else %>$ApplicationName<% end_if %></a></span>
```

Comment:
I think this is ok. It makes sense for pages to link to draft... But your site won't always have a site tree and so on